zkevm-circuits/src/evm_circuit/execution/error_invalid_opcode.rs

```diff
@@ -85,7 +85,7 @@ mod test {
             vec![0xf6],
             vec![0xfe],
             // Multiple invalid opcodes
-            vec![0x5c, 0x5e],
+            vec![0x0c, 0x5e],
```

Comment:
Why is this being changed? 0x5c and 0x0c both seem to be unused opcodes.

Reply:
This is because 0x5c is tload opcode.